src/query/src/range_select/plan.rs

```diff
@@ -322,13 +342,20 @@ impl UserDefinedLogicalNodeCore for RangeSelect {
     fn fmt_for_explain(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         write!(
             f,
-            "RangeSelect: range_exprs=[{}], align={}s time_index={}",
+            "RangeSelect: range_exprs=[{}], align={}s, align_to={}s, align_by=[{}], time_index={}",
             self.range_expr
                 .iter()
                 .map(ToString::to_string)
                 .collect::<Vec<_>>()
                 .join(", "),
             self.align.as_secs(),
+            // convert ms to s
+            self.align_to / 1000,
```

Comment:
It would be better if we print the format in milliseconds as it would help us retain the precision.